packages/dispose_scope/pubspec.yaml

```diff
@@ -1,13 +1,16 @@
 name: dispose_scope
 description: Simple package making it easy to dispose (aka cancel, aka close) your objects.
-version: 2.0.0
+version: 2.0.1
 repository: https://github.com/RobertOdrowaz/dispose-scope/tree/master/packages/dispose_scope
 issue_tracker: https://github.com/RobertOdrowaz/dispose-scope/issues
 homepage: https://github.com/RobertOdrowaz/dispose-scope
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
 
+dependencies:
+  meta: ^1.8.0
```

Comment:
Can we depend on lower version of meta? IIRC flutter_test from flutter 3.3.6 depends on meta 1.7.x so this would make this package virtually unusable in any flutter project

Reply:
Sure 👍